<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainwindow_tba</class>
 <widget class="QDialog" name="mainwindow_tba">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>978</width>
    <height>598</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>30</y>
     <width>771</width>
     <height>501</height>
    </rect>
   </property>
   <property name="title">
    <string>Gestion de client </string>
   </property>
   <widget class="QTabWidget" name="le_idmod">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>741</width>
      <height>451</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="widget">
     <attribute name="title">
      <string>Ajouter</string>
     </attribute>
     <widget class="QLineEdit" name="le_id">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>60</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Id</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>110</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Nom</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>170</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Prenom</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>220</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Numero telephone</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_nom">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>110</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_prenom">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>170</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_numtel">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>220</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>90</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Valider</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>220</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Quitter</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>310</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Homme</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Femme</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>310</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>sexe</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Afficher</string>
     </attribute>
     <widget class="QTableView" name="tab_client">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>421</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>320</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>id</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="recherche_id">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>320</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>supprimer</string>
     </attribute>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>80</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Identifiant</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_id_supp">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>80</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_4">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>250</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>supprimer</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widget_2">
     <attribute name="title">
      <string>Trier</string>
     </attribute>
     <widget class="QPushButton" name="le_trier">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>170</y>
        <width>141</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>trier</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_6">
     <attribute name="title">
      <string>pdf</string>
     </attribute>
     <widget class="QPushButton" name="le_pdf">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>270</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>pdf</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="pdf">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>36</y>
        <width>341</width>
        <height>161</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="widget_3">
     <attribute name="title">
      <string>Statistique</string>
     </attribute>
     <widget class="QPushButton" name="le_stat">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>320</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>statistique</string>
      </property>
     </widget>
     <widget class="QTextEdit" name="stat">
      <property name="geometry">
       <rect>
        <x>133</x>
        <y>56</y>
        <width>361</width>
        <height>161</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>fidelite</string>
     </attribute>
     <widget class="QLineEdit" name="lineEdit_4">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>90</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_13">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>90</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Id</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>150</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Test</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_8">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>90</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QTableView" name="tab_fidelite">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>200</y>
        <width>256</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Alerte</string>
     </attribute>
     <widget class="QLineEdit" name="mail">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>40</y>
        <width>231</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>40</y>
        <width>71</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>adresse mail</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>80</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>objet</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="objet">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>70</y>
        <width>231</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="le_alerte">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>380</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>envoyer</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>160</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>contenu</string>
      </property>
     </widget>
     <widget class="QPlainTextEdit" name="contenu">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>180</y>
        <width>491</width>
        <height>181</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>modifier</string>
     </attribute>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>60</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Id</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_15">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>110</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Nom</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>150</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Prenom</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>200</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Numero telephone</string>
      </property>
     </widget>
     <widget class="QPushButton" name="le_modifier">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>260</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>valider</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_idmod_2">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>60</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_nommod">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>100</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_prenommod">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>140</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="le_numeromod">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>200</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_7">
     <attribute name="title">
      <string>Arduino</string>
     </attribute>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>110</y>
        <width>56</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Lampe</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ON_pushButton_0">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>100</y>
        <width>41</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>ON</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ON_pushButton_1">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>100</y>
        <width>41</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>OFF</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>180</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Etat</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ON_pushButton_2">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>170</y>
        <width>31</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>+</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ON_pushButton_3">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>170</y>
        <width>31</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
